test-vectors/did_dht/create.json

```diff
@@ -0,0 +1,150 @@
+{
+  "description": "did:dht create",
```

Comment:
I would recommend waiting until this PR is merged as the vector will change https://github.com/TBD54566975/did-dht-method/pull/102